src/components/Business.ts

```diff
@@ -0,0 +1,10 @@
+type Business = {
+    id: any;
+    businessName: string;
+    distance: string;
+    futureProofRating: Number;
+    userRating: Number;
+    logoURL: string
+  };
+
+  export default Business
```

Comment:
You can import this from the typings directory using `import {Business} from '@typings/types'`